.github/workflows/publish-draft-release.yml

```diff
@@ -7,13 +7,26 @@ on:
       - v[0-9]+.[0-9]+.[0-9]+*
 
 jobs:
+  get-rust-versions:
+    runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-linux:production
+    outputs:
+      rustc-stable: ${{ steps.get-rust-versions.outputs.stable }}
+      rustc-nightly: ${{ steps.get-rust-versions.outputs.nightly }}
+    steps:
+      - id: get-rust-versions
+        run: |
+          echo "::set-output name=stable::$(rustc +stable --version)"
+          echo "::set-output name=nightly::$(rustc +nightly --version)"
+
   build-runtimes:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         runtime: ['polkadot', 'kusama']
     container:
-      image: paritytech/srtool:nightly-2020-10-27
+      image: chevdor/srtool:nightly-2021-03-15
```

Comment:
according to our [docker images maintenance policy](https://github.com/paritytech/devops/wiki/DevOps-Internal:-Best-Practices:-Docker-images-maintenance-policy) this image should be build in parity ci infrastructure.

Reply:
@chevdor are you OK to ~nightly build and test `srtool` on our infrastructure?

Reply:
Sure thing. Westend does not support the new `on-chain-release-build` feature that is now the default for Kusama and Polkadot. So I need to check whether I add some magic to the feature detection in `srtool` (currently solely based on the version number) or if we bypass the detection and force `BUILD_OPTS` to be empty.

Refs. : 
- https://gitlab.com/chevdor/srtool/-/commit/accf5528a42aff64ee6b0919aa3a16716418fcbd#f6b38ed32919f0ac464cf607efca22951eac98d6_110_78
- https://gitlab.com/chevdor/srtool/-/commit/accf5528a42aff64ee6b0919aa3a16716418fcbd#f6b38ed32919f0ac464cf607efca22951eac98d6_110_78

Reply:
I actually came up with a better way for us (Parity) to handle this, without needing to maintain some Parity-specific fork. I've created a small repo [here](https://gitlab.parity.io/parity/infrastructure/ci_cd/srtool-ci/) with a `.gitlab-ci.yml` that is in charge of building and pushing srtool to *our* Dockerhub, and it has srtool as a git submodule. Whenever we need to update srtool, we can review the changes, and bump the submodule.